sentinelhub/api/batch/base.py

```diff
@@ -36,9 +34,8 @@ class BatchUserAction(Enum):
 
     START = "START"
     ANALYSE = "ANALYSE"
-    NONE = "NONE"
-    CANCEL = "CANCEL"
     STOP = "STOP"
+    NONE = "NONE"
```

Comment:
I only see this one used in the batch statistical notebook, but I don't see it documented in the official docs. Should it be removed?

Reply:
I also didn't see it in the official docs, but when i removed it the tests failed because the enum couldn't parse this value so I added it back in